### DISCUSSION TITLE
How to Hide the Debugging Toolbar and Make It Undetectable

### DISCUSSION THREAD

**Discussion by PankajSavaliya** [seleniumbase/SeleniumBase#2913] [General]
This window continues to show up due to some script running. Is there a way to hide it and make it undetectable? @mdmintz
https://github.com/seleniumbase/SeleniumBase/issues/2404#issuecomment-2211765952

<img width="1273" alt="debugging" src="https://github.com/seleniumbase/SeleniumBase/assets/13134958/aadef920-d183-42e9-93b8-ed540c3f11da">

**Comment by PankajSavaliya** [comment]
@mdmintz Please give me a hint, if possible. It will be very much appreciated.

**Comment by mdmintz** [comment]
Locking this discussion due to https://github.com/seleniumbase/SeleniumBase/issues/2896#issuecomment-2214338488
